### PR TITLE
Refactoring the output

### PR DESCRIPTION
**Pull request by utkarshmani1997**
Separated the code of output into another function  StatsOutput to make it more readable.

### CODE REVIEW

File: command/vsm_stats.go [hunk 1]
```diff
@@ -409,3 +292,135 @@ func GetVolumeStats(address string, obj interface{}) (error, int) {
...
+		err error
+
+		ReadLatency  int64
+		WriteLatency int64
+
+		AvgReadBlockCountPS  int64
+		AvgWriteBlockCountPS int64
+	)
+
+	ReadIOPSi, _ := strconv.ParseInt(stats1.ReadIOPS, 10, 64)
```
Comment (AmitKumarDas) [inline command/vsm_stats.go]: Does the local variable naming follow std golang practices ?

File: command/vsm_stats.go [hunk 2]
```diff
@@ -409,3 +292,135 @@ func GetVolumeStats(address string, obj interface{}) (error, int) {
 	rc := json.NewDecoder(resp.Body).Decode(obj)
 	return rc, 0
 }
+
+// StatsOutput Displays the Output in json or default format
```
Comment (AmitKumarDas) [inline command/vsm_stats.go]: It helps to have better function level comments

File: command/vsm_stats.go (hunk 1 above)
Comment (AmitKumarDas) [inline command/vsm_stats.go]: Its good to comment on why 10, 64 during ParseInt

File: command/vsm_stats.go [hunk 3]
```diff
@@ -409,3 +292,135 @@ func GetVolumeStats(address string, obj interface{}) (error, int) {
...
+	if WriteIOPSPS != 0 {
+		WriteLatency = WriteTimePS / WriteIOPSPS
+		AvgWriteBlockCountPS = WriteBlockCountPS / WriteIOPSPS
+	} else {
+		WriteLatency = 0
+		AvgWriteBlockCountPS = 0
+	}
+
+	SectorSizePS, _ := strconv.ParseFloat(stats2.SectorSize, 64)
+	//	fmt.Println("hi", stats1.SectorSize, stats2.SectorSize)
```
Comment (AmitKumarDas) [inline command/vsm_stats.go]: Delete this.

File: command/vsm_stats.go [hunk 4]
```diff
@@ -409,3 +292,135 @@ func GetVolumeStats(address string, obj interface{}) (error, int) {
...
+
+			IQN:    annotations.Iqn,
+			Volume: args[0],
+			Portal: annotations.TargetPortal,
+			Size:   annotations.VolSize,
+
+			ReadIOPS:  ReadIOPSPS,
+			WriteIOPS: WriteIOPSPS,
+
+			ReadThroughput:  float64(RThroughput) / 104857,
```
Comment (AmitKumarDas) [inline command/vsm_stats.go]: Please associate some typed constants or simple constants for these magic numbers.
Now what is magic number :) Please read

File: command/vsm_stats.go [hunk 5]
```diff
@@ -409,3 +292,135 @@ func GetVolumeStats(address string, obj interface{}) (error, int) {
...
+
+			panic(err)
+		}
+
+		os.Stdout.Write(data)
+
+		fmt.Println("\n------------------------------------")
+
+	} else {
+		fmt.Printf("%7s: %-48s\n", "IQN", annotations.Iqn)
```
Comment (AmitKumarDas) [inline command/vsm_stats.go]: These are magic numbers as well. Please use typed constants or structures to derive meaning & hence maintainability.

File: command/vsm_stats.go [hunk 6]
```diff
@@ -409,3 +292,135 @@ func GetVolumeStats(address string, obj interface{}) (error, int) {
...
+		fmt.Printf("%7s: %-6s\n\n", "Size", annotations.VolSize)
+		fmt.Printf("%s         %s      %s\n", "Replica", "Status", "DataUpdateIndex")
+
+		for i, _ := range statusArray {
+			fmt.Printf("%s\n", statusArray[i])
+		}
+		fmt.Println("------------------------------------")
+
+		// Printing in tabular form
+		w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', tabwriter.AlignRight|tabwriter.Debug)
```
Comment (AmitKumarDas) [inline command/vsm_stats.go]: Magic Numbers Again. Please code in a better way.

**Comment by utkarshmani1997** [comment]
Thanks @AmitKumarDas for the review, i will fix these.

File: command/vsm_stats.go (hunk 1 above)
Comment (utkarshmani1997) [inline command/vsm_stats.go]: @AmitKumarDas variable names should be short, is that what you mean?

**Comment by gila** [comment]
A more generic comment; can we keep the print statements fit in 80 columns?

File: command/vsm_stats.go (hunk 1 above)
Comment (AmitKumarDas) [inline command/vsm_stats.go]: I really do not believe in short variable names as prescribed by golang community.
i.e. https://github.com/golang/go/wiki/CodeReviewComments#variable-names
We can discuss over slack the best practices that we need to follow for coding in any language.

In this comment, I was thinking about camel casing.

**Pull request merged**